### PR TITLE
🌐 Add French translation for `docs/fr/docs/advanced/index.md`

### PR DESCRIPTION
**Pull request by axel584**
Here is the PR to translate the advanced/index.md
See the french translation tracking issue https://github.com/tiangolo/fastapi/issues/1972.

Thanks for the reviews

### CODE REVIEW

File: docs/fr/docs/advanced/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Guide de l'utilisateur avancé - Introduction
+
+## Caractéristiques supplémentaires
+
+Le [Tutoriel - Guide de l'utilisateur](../tutorial/){.internal-link target=_blank} devrait suffire à vous faire découvrir toutes les fonctionnalités principales de **FastAPI**.
+
+Dans les sections suivantes, vous verrez des options, configurations et fonctionnalités supplémentaires.
+
+!!! Conseil
```
Comment (rjNemo) [inline docs/fr/docs/advanced/index.md:9]: until now this was translated has Note

Comment (axel584) [reply docs/fr/docs/advanced/index.md:9]:
```suggestion
!!! Note
```

**Review by rjNemo — APPROVED** [review]

**Review by emmrichard — APPROVED** [review]

**Comment by rjNemo** [comment]
@axel584 You have conflicts to fix

**Review by rjNemo — APPROVED** [review]

**Review by cfraboulet — APPROVED** [review]
LGTM 😊 Well done @axel584

**Comment by tiangolo** [comment]
Nice, thank you @axel584! :cake: 

And thanks for the reviews @emmrichard, @rjNemo, and @frabc :coffee:

**Pull request merged**